### ISSUE TITLE
{SPIKE} Should components provide haptic or audio feedback?

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#3132]
### Overview
The [HIG indicates that haptic and audio feedback should be considered for custom UI components](https://developer.apple.com/ios/human-interface-guidelines/user-interaction/feedback/). Material Guidelines also indicate that [non-visual feedback should be provided](https://material.io/guidelines/usability/accessibility.html#accessibility-sound-motion) to accommodate users who cannot rely on visual cues. This may include affordances for VoiceOver, but also mentions audio feedback. One notable exception: "Extra sounds added to native elements."

Should Material Components for iOS provide non-visual feedback for interaction? 

---
#### Internal data
- Associated internal bug: [b/117179243](http://b/117179243)

**Comment by material-mirror** [comment]
The internal issue [b/117179243](http://b/117179243) is now closed. This issue is being closed as a result.
